odoo/addons/base/wizard/base_language_install.py

```diff
@@ -38,7 +38,6 @@ def lang_install(self):
         mods = self.env['ir.module.module'].search([('state', '=', 'installed')])
         self.lang_ids.active = True
         mods._update_translations(self.lang_ids.mapped('code'), self.overwrite)
-        self.env.cr.execute('ANALYZE ir_translation')
```

Comment:
Yes, it is on schedule

Reply:
This was added as a fix to a performance issue on the runbot.  I think that it is less of a problem now, since the translations end up in various tables which are much smaller in size than `ir.translation`.